modules/modifying-an-existing-ingress-controller.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// *ingress-controller-dnsmgt.adoc
+
+:_content-type: PROCEDURE
+[id="modifying-an-existing-ingress-controller_{context}"]
+= Modifying an existing Ingress Controller
+
+As a cluster administrator, you can modify an existing Ingress Controller to manage DNS record lifecycle.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+* DNS record created.
```

Comment:
```suggestion
* A DNS record is created and published.
```

Reply:
I think this pre-req is not required when creating an IG CR

Reply:
Since we are modifying an existing ing controller to set as `Unmanaged`, we would still need an externally created DNS record.